poll/poll.py

```diff
@@ -1163,7 +1167,7 @@ def vote(self, data, suffix=''):
 
         # Make sure the answer values are sane.
         for key, value in data.items():
-            if value not in answers.keys():
+            if value not in list(answers.keys()):
```

Comment:
Why are we changing this? Don't we want to use `dict_keys` in Python 3?

Reply:
Fixed. I remember running test for python 3 was raising issues earlier. Anyhow i have reverted this change.